src/components/Profile.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom';
+
+class Profile extends Component {
+  render() {
+    return (
+      <form>
```

Comment:
Not really sure why the profile info is in a form when it can't be edited.  Maybe that's intended for a future version?